--- origsrc/gtkhtml-4.2.2/components/editor/Makefile.am	2011-10-20 05:31:32.000000000 -0500
+++ src/gtkhtml-4.2.2/components/editor/Makefile.am	2011-11-16 18:05:35.385339600 -0600
@@ -134,7 +134,7 @@ libglade_gtkhtml_editor_la_SOURCES = \
 
 libglade_gtkhtml_editor_la_LDFLAGS = -module -avoid-version -no-undefined
 
-libglade_gtkhtml_editor_la_LIBADD = @GTKHTML_CFLAGS@
+libglade_gtkhtml_editor_la_LIBADD = @GTKHTML_LIBS@
 
 endif
 
